toolvoice/scripts/evaluation/evaluate_confetti.py

```
@@ -0,0 +1,998 @@
+            for func_name, params in item.items():
+                if isinstance(params, dict):
+                    parameters = [ParameterNode(name=k, value=v) for k, v in params.items()]
+                else:
+                    parameters = []
+                function_calls.append(FunctionCallNode(name=func_name, parameters=parameters))
+
+    return function_calls
+
+
+def match_function_calls(predicted: FunctionCallNode, gold: FunctionCallNode,
+                         use_soft_matching: bool = False) -> MatchResult:
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)